database/seeders/AdminsTableSeeder.php

```diff
@@ -18,7 +18,7 @@ public function run()
             //Name, email, password, type
             'name'     => 'Alex Legard',
             'email'    => 'alexlegard3@gmail.com',
```

Comment:
Personally I wouldn't put your email address in the actual codebase to prevent bots and stuff from starting to spam you.

Reply:
I could go env('whatever') instead.